.github/workflows/CI.yml

```diff
@@ -19,8 +19,9 @@ jobs:
         env:
```

Comment:
Please do not forget that the same changes added here have to be mirrored to CD file.

Reply:
It seems to me that this comment is still relevant.

Reply:
I read and successfully forgot about it